### DISCUSSION TITLE
weird afterStateUpdated behavior

### DISCUSSION THREAD

**Discussion by ALSHIHAB** [filamentphp/filament#8809] [Help] [answered]
I've used afterStateUpdated in V2 and it was great. In v3 Ive used it in creating slug from product name with live() and it is so noisy that whenever I keyin product name it is slow and need to wait few seconds between keyin letters otherwise ignores many letters!!!

**Comment by leandrocfe** [comment, marked as answer]
You could use ->live(onBlur:true)
https://filamentphp.com/docs/3.x/forms/advanced/#reactive-fields-on-blur

**Reply by ALSHIHAB** [reply]
Thanks

**Reply by nepriantodeveloper** [reply]
Thanks
